Core/TImeSeries/CumulativeTimeSeries.cs

```diff
@@ -0,0 +1,56 @@
+using System.Collections.Generic;
+using System;
+using System.Linq;
+using Core;
+
+namespace Core.TimeSeries
+{
+	public sealed class CumulativeTimeSeries : TimeSeries
+	{
+        private List<double> AccumulatedSeries { get; } = new List<double>();
+        public double Increment { get; }
+        public double Capacity { get; }
+        
+        public CumulativeTimeSeries(double increment, double capacity)
+        {
+            Increment = increment;
+            Capacity = capacity;
+        }
+
+        public CumulativeTimeSeries(List<Transaction> transactions, double increment, double capacity) : this(increment, capacity) =>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this constructor to use a generic collection designed for inheritance.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4992784)